ketchup/main.py

```diff
@@ -1,9 +1,9 @@
 import datetime
-
+import math
 import dateparser
 import datetime as dt
 from ketchup.luffy import LUFFY
-import ketchup.watch_schedule_builder as wsb
+import ketchup.watch_schedule_builder as dc
```

Comment:
i think you want to change this back to wsb

Reply:
Yeah, merge errors came through. Will resolve.